trino/base/hive-config.yaml

```diff
@@ -167,7 +167,7 @@ data:
     \    <value>false</value>\n  </property>\n  <property>\n    <name>hive.metastore.schema.verification</name>\n\
     \    <value>false</value>\n  </property>\n  <property>\n    <name>hive.default.fileformat</name>\n\
     \    <value>Parquet</value>\n  </property>\n  <property>\n    <name>hive.metastore.warehouse.dir</name>\n\
-    \    <value>s3a://XXX_S3ENDPOINT_XXX/</value>\n  </property>\n        <property> <name>fs.s3a.endpoint</name>\n\
+    \    <value>/opt/hive/warehouse</value>\n  </property>\n        <property> <name>fs.s3a.endpoint</name>\n\
```

Comment:
If we want this to persist, should we mount it on a PVC?

Reply:
I read in Hive docs that warehouse dir doesn't store anything relevant, so al data should be in S3 and metadata in postgresql. Having that said, mounting this path in a pvc would be useless.